spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -40,39 +33,98 @@
                                           }
 
       expect(response.status).to eq 201
-      expect(json_response_created_skill_name).to eq("created")
-      expect(json_response_created_skill_featured).to eq(true)
-      expect(json_response_created_skill_group_id).to eq(group.id)
+      expect(json_response_skill_name).to eq("created")
+      expect(json_response_skill_featured).to eq(true)
+      expect(json_response_skill_group_id).to eq(group.id)
     end
 
     it "responds with error messages if a skill fails to create" do
       post :create, format: :json, skill: { name: "" }
 
       expect(response.status).to eq 422
-      expect(json_response_uniquenss_error_message).to eq(["Name can't be blank"])
+      expect(json_response_error_message).to eq(["Name can't be blank"])
+    end
+  end
+
+  describe "show" do
+
+    it "returns an individual skill" do
+      skill = create(:skill, name: "show_skill")
+
+      get :show, format: :json, id: skill.id
+
+      expect(response.status).to eq 200
+      expect(json_response_skill_name).to eq("show_skill")
+    end
+  end
+
+  describe "update" do
+
+    it "updates a skill" do
+      group_1 = create(:group, name: "before_updated_group")
+      group_2 = create(:group, name: "after_updated_group")
+      skill = create(:skill, name: "before_updated_name", group: group_1)
+
+      put :update, format: :json, 
+                   id: skill.id, 
+                   skill: {
+                            name: "after_updated_name",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.